[2025-06-13T11:05:03.205+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-06-13T11:05:03.233+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: hourly_clickhouse_metrics_dag.extract_clickhouse_metrics scheduled__2025-06-13T10:00:00+00:00 [queued]>
[2025-06-13T11:05:03.242+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: hourly_clickhouse_metrics_dag.extract_clickhouse_metrics scheduled__2025-06-13T10:00:00+00:00 [queued]>
[2025-06-13T11:05:03.243+0000] {taskinstance.py:2306} INFO - Starting attempt 2 of 2
[2025-06-13T11:05:03.257+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): extract_clickhouse_metrics> on 2025-06-13 10:00:00+00:00
[2025-06-13T11:05:03.267+0000] {standard_task_runner.py:63} INFO - Started process 645 to run task
[2025-06-13T11:05:03.271+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'hourly_clickhouse_metrics_dag', 'extract_clickhouse_metrics', 'scheduled__2025-06-13T10:00:00+00:00', '--job-id', '30', '--raw', '--subdir', 'DAGS_FOLDER/hourly_clickhouse_metrics_dag.py', '--cfg-path', '/tmp/tmp2p0t6fo3']
[2025-06-13T11:05:03.275+0000] {standard_task_runner.py:91} INFO - Job 30: Subtask extract_clickhouse_metrics
[2025-06-13T11:05:03.496+0000] {task_command.py:426} INFO - Running <TaskInstance: hourly_clickhouse_metrics_dag.extract_clickhouse_metrics scheduled__2025-06-13T10:00:00+00:00 [running]> on host 85176eedf8e8
[2025-06-13T11:05:03.543+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.10/site-packages/***/utils/context.py:212: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2025-06-13T11:05:03.579+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='hourly_clickhouse_metrics_dag' AIRFLOW_CTX_TASK_ID='extract_clickhouse_metrics' AIRFLOW_CTX_EXECUTION_DATE='2025-06-13T10:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-06-13T10:00:00+00:00'
[2025-06-13T11:05:03.579+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-06-13T11:05:03.608+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-06-13T11:05:03.608+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/scripts/extract_clickhouse_metrics.py", line 24, in extract_and_save_metrics
    data = client.execute(query)
AttributeError: 'HttpClient' object has no attribute 'execute'
[2025-06-13T11:05:03.618+0000] {taskinstance.py:1206} INFO - Marking task as FAILED. dag_id=hourly_clickhouse_metrics_dag, task_id=extract_clickhouse_metrics, run_id=scheduled__2025-06-13T10:00:00+00:00, execution_date=20250613T100000, start_date=20250613T110503, end_date=20250613T110503
[2025-06-13T11:05:03.629+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 30 for task extract_clickhouse_metrics ('HttpClient' object has no attribute 'execute'; 645)
[2025-06-13T11:05:03.683+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2025-06-13T11:05:03.869+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-06-13T11:05:03.870+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2025-06-13T11:33:14.941+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-06-13T11:33:14.964+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: hourly_clickhouse_metrics_dag.extract_clickhouse_metrics scheduled__2025-06-13T10:00:00+00:00 [queued]>
[2025-06-13T11:33:14.972+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: hourly_clickhouse_metrics_dag.extract_clickhouse_metrics scheduled__2025-06-13T10:00:00+00:00 [queued]>
[2025-06-13T11:33:14.972+0000] {taskinstance.py:2306} INFO - Starting attempt 2 of 2
[2025-06-13T11:33:14.985+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): extract_clickhouse_metrics> on 2025-06-13 10:00:00+00:00
[2025-06-13T11:33:14.993+0000] {standard_task_runner.py:63} INFO - Started process 152 to run task
[2025-06-13T11:33:15.001+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'hourly_clickhouse_metrics_dag', 'extract_clickhouse_metrics', 'scheduled__2025-06-13T10:00:00+00:00', '--job-id', '6', '--raw', '--subdir', 'DAGS_FOLDER/hourly_clickhouse_metrics_dag.py', '--cfg-path', '/tmp/tmpn_igs5js']
[2025-06-13T11:33:15.005+0000] {standard_task_runner.py:91} INFO - Job 6: Subtask extract_clickhouse_metrics
[2025-06-13T11:33:15.226+0000] {task_command.py:426} INFO - Running <TaskInstance: hourly_clickhouse_metrics_dag.extract_clickhouse_metrics scheduled__2025-06-13T10:00:00+00:00 [running]> on host 88ee0e123f4a
[2025-06-13T11:33:15.290+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.10/site-packages/***/utils/context.py:212: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2025-06-13T11:33:15.333+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='hourly_clickhouse_metrics_dag' AIRFLOW_CTX_TASK_ID='extract_clickhouse_metrics' AIRFLOW_CTX_EXECUTION_DATE='2025-06-13T10:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-06-13T10:00:00+00:00'
[2025-06-13T11:33:15.334+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-06-13T11:33:15.376+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-06-13T11:33:15.377+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/scripts/extract_clickhouse_metrics.py", line 24, in extract_and_save_metrics
    data = client.execute(query)
AttributeError: 'HttpClient' object has no attribute 'execute'
[2025-06-13T11:33:15.388+0000] {taskinstance.py:1206} INFO - Marking task as FAILED. dag_id=hourly_clickhouse_metrics_dag, task_id=extract_clickhouse_metrics, run_id=scheduled__2025-06-13T10:00:00+00:00, execution_date=20250613T100000, start_date=20250613T113314, end_date=20250613T113315
[2025-06-13T11:33:15.400+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 6 for task extract_clickhouse_metrics ('HttpClient' object has no attribute 'execute'; 152)
[2025-06-13T11:33:15.449+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2025-06-13T11:33:15.648+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-06-13T11:33:15.649+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
